catalog-info.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: avalanche-cli
+  description: |
+    Avalanche-CLI is a command-line interface for building, deploying, and maintaining Avalanche
```

Comment:
do we need duplication of this?

Reply:
can we just have on root level yml? (sorry if ignorant on this kind of conf)

Reply:
sadly we have to duplicate here